src/WebJobs.Script/Settings/ISettingsManager.cs

```diff
@@ -0,0 +1,12 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+namespace Microsoft.Azure.WebJobs.Script.Settings
```

Comment:
Rather than a new folder/namespace, lets this to the existing Config folder/namespace, which is where all configuration things go